superset-frontend/packages/superset-ui-core/src/style/index.tsx

```diff
@@ -59,14 +59,24 @@ const defaultTheme = {
       label: '#879399',
       help: '#737373',
     },
+    grayscale: {
+      base: '#666666',
+      dark1: '#323232',
+      dark2: '#000000',
+      light1: '#B2B2B2',
+      light2: '#F0F0F0',
+      light3: '#F7F7F7',
+      light4: '#FFFFFF',
```

Comment:
Is it intentional that light4 and light5 are now equal? Same for primary and secondary

Reply:
oops I think I had spillover / PR overflow from my larger theming PR, let me take this out

Reply:
was trying to align the brigthness levels across named colors, bailing on this and adding logic in the theme to point to the right share....